common/laws/00_free_speech.txt

```diff
@@ -156,7 +158,7 @@ law_protected_speech = {
 	
 	modifier = {
 		country_tech_spread_mult = 0.20
-		country_institution_multiculturalism_max_investment_add = 1
+		state_radicals_from_sol_change_mult = 0.05
```

Comment:
This means you get 5% more radicals, intended?

Reply:
Yes, was meant to be a penalty for protecting speech which gives that massive 20% tech spread boost and leads down the path of a liberal multicultural society

Reply:
Hmm, sounds fair :)

Aight, approved!